### DISCUSSION TITLE
Don't stop container if coordinator goes offline

### DISCUSSION THREAD

**Discussion by andreapx** [Koenkk/zigbee2mqtt#19311] [General]
Hi!
I have a SMLIGHT SLZB-06 coordinator (ethernet) and whenever it gets disconnected, the Zigbee2MQTT container stops. Is there a way to not stop it and make it try to connect again and again to the coordinator?
In case of power failure the server stays on because it's on a UPS, but the coordinator is not, so I have to manually restart the container (not a big problem in my house, but kind of a problem at my parents).

**Comment by degreal** [comment]
Exactly the same problem here. Server is in the basement behind the UPS but coordinator is upstairs where the best signal is. In case of power outage or network problems zigbee2mqtt addon stops and never recovers. Very fragile.

**Comment by darkpixel** [comment]
Yeah, I was just wondering about that too.
We had a brief power outage due to storms, and it seems like the default action for Zigbee2MQTT is "there was the briefest interruption, so I'm just going to die and never come back until you manually intervene".

It seems like it should stay running and continue attempting to contact the coordinator.

**Reply by chris-1243** [reply]
I would use the watchdog

https://www.zigbee2mqtt.io/guide/installation/15_watchdog.html#watchdog

**Reply by darkpixel** [reply]
*facepalm*
Yeah...that makes sense. ;)
